src/components/HistoryList.vue

```diff
@@ -0,0 +1,69 @@
+<template>
+  <div class="list"> 
+    <div class="title">{{ title }}</div>
+    <div class="elements">
+      <template v-for="item in items">
+        <div v-bind:key="item.time.toString()" class="element">
+          <div>{{ item.time | date }}<a :href="'https://explorer.avax-test.network/tx/' + item.tx" target="_blank"><i class="fa fa-external-link" aria-hidden="true"></i></a></div>
```

Comment:
I would move the logic of the link generation to a separate module (e.g. blockchain.js) and use it in this component through a method. It will help in future to avoid the code duplication problems